test/lint/lint-spelling.ignore-words.txt

```diff
@@ -1,17 +1,17 @@
 hights
 mor
-mut
-objext
 useable
 wit
 unparseable
-copyable
 cachable
-errorstring
-keyserver
-homogenous
-setban
 hist
 ser
-unselect
-lowercased
+asend
+atack
```

Comment:
Is ignoring the `contrib/gitian-keys/keys.txt` file a more reliable and future-proof solution?

Reply:
Agreed that it would be better to exclude files from the spelling linter that merely consists of names. Maybe a follow-up PR would be appropriate here, where not only the gitian keys file, but all other candidates are thoroughly identified and excluded?

Reply:
Why a separate pr for a one-line change?

```diff
--- a/test/lint/lint-spelling.sh
+++ b/test/lint/lint-spelling.sh
@@ -15,6 +15,6 @@ if ! command -v codespell > /dev/null; then
 fi
 
 IGNORE_WORDS_FILE=test/lint/lint-spelling.ignore-words.txt
-if ! codespell --check-filenames --disable-colors --quiet-level=7 --ignore-words=${IGNORE_WORDS_FILE} $(git ls-files -- ":(exclude)build-aux/m4/" ":(exclude)contrib/seeds/*.txt" ":(exclude)depends/" ":(exclude)doc/release-notes/" ":(exclude)src/leveldb/" ":(exclude)src/crc32c/" ":(exclude)src/qt/locale/" ":(exclude)src/qt/*.qrc" ":(exclude)src/secp256k1/" ":(exclude)src/univalue/"); then
+if ! codespell --check-filenames --disable-colors --quiet-level=7 --ignore-words=${IGNORE_WORDS_FILE} $(git ls-files -- ":(exclude)build-aux/m4/" ":(exclude)contrib/seeds/*.txt" ":(exclude)depends/" ":(exclude)doc/release-notes/" ":(exclude)src/leveldb/" ":(exclude)src/crc32c/" ":(exclude)src/qt/locale/" ":(exclude)src/qt/*.qrc" ":(exclude)src/secp256k1/" ":(exclude)src/univalue/" ":(exclude)contrib/gitian-keys/keys.txt"); then
     echo "^ Warning: codespell identified likely spelling errors. Any false positives? Add them to the list of ignored words in ${IGNORE_WORDS_FILE}"
 fi
```

Reply:
I think this does strictly speaking more than the PR title suggests, but OTOH I don't mind adding it. (Though, for this specific case, considering jonatacks high involvement and activity in the project though, it's quite likely that his last name will appear in some other file (e.g. in doc/) and it will have to end up in the spelling linter ignorelist again :))

Anyways, added another commit. If it creates controversy for some reason, it can easily be removed from the PR without invalidating the previous ACKs.